app/containers/Passcode/utils.ts

```
@@ -1,14 +1,15 @@
-import AsyncStorage from '@react-native-async-storage/async-storage';
+import UserPreferences from '../../lib/methods/userPreferences';
```

ESLint

[error] There should be no empty line within import group

(import/order)